vars/sfdxBuildPipeline.groovy

```diff
@@ -150,6 +180,33 @@ def call(Map parameters = [:]) {
             stage("publish") {
                 junit keepLongStdio: true, testResults: 'tests/**/*-junit.xml'
             }
+
+            if (notificationChannel?.trim()) {
+                stage("slack notification end") {
+                    /*
+                    * Slack integration
+                    * https://api.slack.com/docs/message-guidelines
```

Comment:
Does this link explain messages colours? The link I can explaining about overall slack API is at https://api.slack.com/start and the Jenkins Slack plugin doc is at https://plugins.jenkins.io/slack/.

Reply:
Full comment added (from Slack doc) and URL updated.